src/pages/directory/directory.module.ts

```diff
@@ -1,11 +1,19 @@
 import { NgModule } from '@angular/core';
-import { SearchPipe } from '../../pipes/search/search';
+import { IonicPageModule } from 'ionic-angular';
+
+import { DirectoryPage } from './directory';
+import { PipesModule } from '../../pipes/pipes.module';
 
 @NgModule({
   declarations: [
-    SearchPipe
+    DirectoryPage
   ],
   imports: [
+    IonicPageModule.forChild(DirectoryPage),
```

Comment:
This is how to register a sub-module with Ionic so it knows what page to render when it needs to get lazy-loaded.